tools/python/run_CIs_for_external_pr.py

```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/env python3
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+import argparse
+import os
+import subprocess
+import sys
+import typing
+
+parser = argparse.ArgumentParser(
+    os.path.basename(__file__),
+    formatter_class=argparse.RawDescriptionHelpFormatter,
+    description="""Trigger CIs running for the specified pull request.
+
+    Requires the GitHub CLI to be installed. See https://github.com/cli/cli#installation for details.
+    After installation you will also need to setup an auth token to access the ONNX Runtime repository by running
+    `gh auth login`. Easiest is to run that from a directory in your local copy of the repo.
+    """,
+)
+parser.add_argument("--pr", required=True, help="Specify the pull request ID.")
```

Comment:
I would keep the program in a main() function: https://google.github.io/styleguide/pyguide.html#317-main